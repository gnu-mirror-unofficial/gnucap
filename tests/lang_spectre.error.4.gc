
spectre

// duplicate label error
subckt crash (n0 n1 n2 n3 n0 n4)
ends
subckt zero (n0 n1 n2 n3 0 n4)
ends

X1 (1 2) crash
- list
- op
- end
