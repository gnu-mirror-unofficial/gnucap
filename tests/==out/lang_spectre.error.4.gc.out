subckt crash (n0 n1 n2 n3 n0 n4) 
                          ^ ? duplicate port name, skipping
subckt zero (n0 n1 n2 n3 0 n4) 
                         ^ ? node 0 not allowed here
// duplicate label error 
subckt crash (n0 n1 n2 n3 n4)
ends crash

subckt zero (n0 n1 n2 n3 n4)
ends zero

X1 (1 2) crash 
#          
 27.       
